### PR TITLE
read container tags from env vars

### PR DESCRIPTION
**Pull request by lujomon**

### CODE REVIEW

File: plugins/outputs/sematext/processors/container.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+	tags[K8S_POD_ENV_NAME] = os.Getenv(K8S_POD_NAME_TAG)
+	tags[K8S_NAMESPACE_ENV_NAME] = os.Getenv(K8S_NAMESPACE_ID_TAG)
+	tags[K8S_CLUSTER_ENV_NAME] = os.Getenv(K8S_CLUSTER_TAG)
+	return &ContainerTags{
+		tags: tags,
+	}
+}
+
+// Process is a method where Heartbeat processor checks whether a heartbeat metric is needed and injects it if so
```
Comment (bsmid) [inline plugins/outputs/sematext/processors/container.go]: Comment belongs to other processor.

File: plugins/outputs/sematext/processors/container.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+type ContainerTags struct {
+	tags map[string]string
+}
+
+// NewContainerTags creates new instance of container MetricProcessor
+func NewContainerTags() MetricProcessor {
+	tags := make(map[string]string)
+	tags[CONTAINER_NAME_TAG] = os.Getenv(CONTAINER_NAME_ENV_NAME)
+	tags[CONTAINER_ID_TAG] = os.Getenv(CONTAINER_ID_ENV_NAME)
+	tags[CONTAINER_IMAGE_NAME_TAG] = os.Getenv(CONTAINER_IMAGE_NAME_TAG)
```
Comment (bsmid) [inline plugins/outputs/sematext/processors/container.go]: Same const on both sides.

File: plugins/outputs/sematext/processors/container.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+	tags[CONTAINER_IMAGE_TAG_ENV_NAME] = os.Getenv(CONTAINER_IMAGE_TAG_TAG)
```
Comment (bsmid) [inline plugins/outputs/sematext/processors/container.go]: Consts on the left and right side should be reversed. Same in a few other cases below.

Comment (lujomon) [reply plugins/outputs/sematext/processors/container.go]: huh, that was embarrasing. Pushed fix.

**Review by bsmid — APPROVED** [review]

**Pull request merged**